.docker/docker_run_git.sh

```diff
@@ -113,6 +113,22 @@ if [ $PS_DEMO_MODE -ne 0 ]; then
     sed -ie "s/define('_PS_MODE_DEMO_', false);/define('_PS_MODE_DEMO_',\ true);/g" /var/www/html/config/defines.inc.php
 fi
 
+if [ $BLACKFIRE_ENABLE -eq 1 ]; then
+    if [ "$BLACKFIRE_SERVER_ID" = "0" ] || [ "$BLACKFIRE_SERVER_TOKEN" = "0" ]; then
+            echo "\n* BLACKFIRE_SERVER_ID and BLACKFIRE_SERVER_TOKEN environment variables missing."
+            echo "\n* Skipping blackfire install..."
+    else
+      echo "\n* Installing Blackfire..."
+      wget -q -O - https://packages.blackfire.io/gpg.key | dd of=/usr/share/keyrings/blackfire-archive-keyring.asc
```

Comment:
On develop we use curl to recover nvm, we might as well use the same tool right? This would avoid having to have both https://github.com/PrestaShop/PrestaShop/blob/dd60d034b6586c1601db7df46162c282bbf308bb/.docker/docker_run_git.sh#L44